kindcat_custom.cue

```diff
@@ -130,19 +140,19 @@ Custom: S={
 		// This field could be inlined into `group`, but is separate for clarity.
 		_computedGroups: [
 			if S.crd.groupOverride != _|_ {
-				strings.ToLower(S.crd.groupOverride) + ".apps.grafana.com",
+				strings.ToLower(S.crd.groupOverride) + ".ext.grafana.com",
```

Comment:
@IfSentient as per our discussion earlier this week, let's make sure there's an escape hatch of overriding also the group suffix, in case there's no viable migration path for SLO / CLE apps.

Reply:
Yup, most recent commit should have `groupOverride` overriding the whole group string.